Add cc as personal document type for PT

BaltaRT25 opened the pull request:
### Context
Currently, `nif` is used as tax document type and as personal document type. This has changed, so `cc` (Cartão de cidadão) must be added as personal document type and `nif` is only accepted as tax document type.

Added a function to validate the `cc` value as a new personal document type. The `cc` value can only be 8 digits long.

BaltaRT25 committed:
New personal document type

Added changes to accept "cc" as personal document type instead of "nif". The nif value only will be accepted as tax document type.

BaltaRT25 committed:
Validate cc value

Added a function to validate the "cc" value as a new personal document type. The "cc" value can only be 8 digits long.

BaltaRT25 committed:
Unit testing

Added unit test

erickgnavar commented on test/bali_test.exs:
```diff
@@ -267,12 +282,12 @@ defmodule BaliTest do
-    assert {:ok, "123456789"} == Bali.validate_personal_document(:pt, :nif, "123456789")
+  test "Puedo validar que el documento personal para Portugal(cc) pertenece al conjunto de documentos válidos y su valor es correcto" do
+    assert {:ok, "12345678"} == Bali.validate_personal_document(:pt, :cc, "12345678")
   end
 
-  test "Puedo validar que el documento personal para Portugal(invalid_nif) no pertenece al conjunto de documentos válidos" do
+  test "Puedo validar que el documento personal para Portugal(invalid_cc) no pertenece al conjunto de documentos válidos" do
     assert {:error, "Documento personal inválido para el país: pt"} ==
-             Bali.validate_personal_document(:pt, :invalid_nif, "123456789")
+             Bali.validate_personal_document(:pt, :invalid_cc, "123456789")
```
Si estos tests son de `nif` que no se ha eliminado sino solo se está agregando un nuevo tipo, ¿por qué se editan en lugar de agregar nuevos?

BaltaRT25 replied:
Porque `nif` se eliminó de documentos personales de PT y su test para probar un `invalid_nif` de documento fiscal está por [acá](https://github.com/resuelve/bali/blob/5a011614507def76652777ebd0c9e30e379ddcfc/test/bali_test.exs#L201C52-L201C73)

erickgnavar reviewed (approved):
🦭

BaltaRT25 committed:
NIF and CC as personal document

Added nif and cc as personal document type. More details: https://app.shortcut.com/gobravo/story/58871/pt-campo-cc-carta-poder#activity-59752

poncho reviewed (approved):
🦭

erickgnavar commented:
Consideren que este repo es público, no  pongan cosas de negocio que son privadas
